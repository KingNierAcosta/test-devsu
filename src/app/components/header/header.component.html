<header class="">
  <img class="w-56" src="/assets/images/logo.png" alt="logo" />
</header>
