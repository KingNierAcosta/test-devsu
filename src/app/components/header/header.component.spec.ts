import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
import { RouterTestingModule } from '@angular/router/testing';

import { HeaderComponent } from './header.component';

describe('HeaderComponent', () => {
  let component: HeaderComponent;
  let fixture: ComponentFixture<HeaderComponent>;
  let compiled: HTMLElement;

  beforeEach(waitForAsync(() => {
    TestBed.configureTestingModule({
      imports: [HeaderComponent, RouterTestingModule],
    }).compileComponents();
  }));

  beforeEach(async () => {
    fixture = TestBed.createComponent(HeaderComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should match the snapshot', () => {
    expect(compiled).toMatchSnapshot();
  });
});
