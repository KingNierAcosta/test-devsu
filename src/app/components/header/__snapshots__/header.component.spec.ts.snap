// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should match the snapshot 1`] = `
<div
  id="root1"
>
  <header>
    <img
      alt="logo"
      class="w-56"
      routerlink="/products"
      src="/assets/images/logo.png"
      tabindex="0"
    />
  </header>
</div>
`;
